### ISSUE TITLE
Admin new order emails do not have edit link or date

### ISSUE THREAD

**Issue by claudiulodro** [woocommerce/woocommerce#17165]
3.1 version has link to edit order at top and the order date:
<img width="386" alt="screen shot 2017-10-12 at 8 41 01 am" src="https://user-images.githubusercontent.com/7317227/31505328-81c41a38-af29-11e7-8254-c73ee16e81db.png">

3.2 version has no link or date at top:
<img width="382" alt="screen shot 2017-10-12 at 8 41 31 am" src="https://user-images.githubusercontent.com/7317227/31505329-81ddc2ee-af29-11e7-9394-19c6bb08af2a.png">

The admin email should have those fields at the top of the email because they are handy.

**Comment by lorrinevil** [comment]
thanks - have several bugs and was coming to look for this fix - on to the next one

**Comment by nitaserban** [comment]
please please please solve it as soon as you can, because it causes big issues on cheking the order via mail.

thank you in advance!

**Comment by claudiulodro** [comment]
@nitaserban It's solved at #17170. You can apply that fix yourself if you are comfortable doing that, otherwise it will be in 3.2.1 which will be releasing next week.

**Comment by nitaserban** [comment]
Thank you for your concern and answer. I tried last night and seemed to work. I am new , ground level in PHP & coding and i was a bit scared not to mess up scripts. I compared the 3.1 and 3.2 email-order-items.php, took the code part missing from email-order-items.php from 3.1 and put it in 3.2 and simply worked. :) Told you, am i ground zero in PHP, did not understand what i was doing at the moment, but what could go wrong if i backed up the original file?!  Thanks again. Look forward dor the update.
